soundbay/conf/main.yaml

```diff
@@ -0,0 +1,8 @@
+# @package _global_
```

Comment:
Why is this file exist?
Should there be so many new `yaml` files?

Reply:
I think that different types of experiments (e.g. with many different parameters such as model, preprocessing, optimizer, and not just one different parameter) are easer to handle when they have their own main. If it's only one parameter like a dataset path it's simple enough to add an override to the train command line.  The new "optim", "preprocessors" and "model" yamls are of course required.

But on a related note, I may have misunderstood the changes you made in the latest PR to the hierarchy, I thought that every "runs" conf file should now be in the parent folder in order to reach all the relevant yamls, so I moved main, main_inference and the parallel files for the googlenet manatees experiment up to the conf folder.